testfiles/CMakeLists.txt

```diff
@@ -695,6 +695,7 @@ add_simulation_test(IDF_FILE _1Zone_Heavy_SelfRef.idf EPW_FILE USA_CO_Golden-NRE
 add_simulation_test(IDF_FILE _5ZoneEvapCooled.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE _AllOffOpScheme.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE _BranchPumpsWithCommonPipe.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+add_simulation_test(IDF_FILE _CrashLights_9680.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
```

Comment:
Before fix:

```
   ** Severe  ** GetInternalHeatGains: Lights="HEAT EXCHANGER AIR TO AIR SENSIBLE AND LATENT 1", invalid Heat Exchanger Type = PLATE
   **   ~~~   ** No matching Zone Exhaust Air Node found.
   **  Fatal  ** GetInternalHeatGains: Errors found in Getting Internal Gains Input, Program Stopped
```

Reply:
Confirmed this error in develop running this file, and success in this branch.  :+1: